dotnet-desktop-guide/framework/wpf/controls/snippets/popup-overview/csharp/Window1.xaml.cs

```diff
@@ -36,20 +36,20 @@ private void setColors(object sender, RoutedEventArgs e)
         }
         //</SnippetIsFocused>
 
-        //<SnippetCreatePopupCode>
+        //<CreatePopup>
```

Comment:
Is this snippet used anywhere? I don't see it any references to it so just want to make sure I didn't miss anything. Other than that, looks good to me.

Reply:
No, just old copy\paste